test/functional/p2p-fullblocktest.py

```diff
@@ -18,6 +18,7 @@
 import time
 from test_framework.key import CECKey
 from test_framework.script import *
+from test_framework.script import hash160
```

Comment:
Yes, I agree that they should all live in the same place. My preference would be for utils.py to actually be pure utility functions and for the hash functions to live in there.

I haven't moved it in this commit because that seemed the minimal change. We should move hash160 to live alongside the other functions in a future PR that cleans up utils.py and mininode.py.